crates/router/src/connector/utils.rs

```diff
@@ -132,12 +132,56 @@ impl PaymentsRequestData for types::PaymentsAuthorizeRouterData {
 
 pub trait PaymentsAuthorizeRequestData {
     fn is_auto_capture(&self) -> bool;
+    //currently amount will be in cents, this method will convert the amount to dollars Eg: 991 cents to 9.91 dollars
+    fn get_amount_in_dollars(&self) -> String;
 }
 
 impl PaymentsAuthorizeRequestData for types::PaymentsAuthorizeData {
     fn is_auto_capture(&self) -> bool {
         self.capture_method == Some(storage_models::enums::CaptureMethod::Automatic)
     }
+    fn get_amount_in_dollars(&self) -> String {
+        #[allow(clippy::as_conversions)]
+        (self.amount as f32 / 100.0).to_string()
+    }
+}
+
+pub trait PaymentsSessionRequestData {
+    //By default amount will be in cents, this method will convert the amount to dollars. Eg: 991 cents to 9.91 dollars
+    fn get_amount_in_dollars(&self) -> String;
```

Comment:
We support all currencies. Not only Dollars. By definition itself it will break for other currencies.

How other currencies are handled?